_docs/concepts/service-model.md

```diff
@@ -3,35 +3,18 @@ category: Concepts
 title: Service Model
 
 parent: Traffic Management
-order: 0
+order: 20
 
 bodyclass: docs
 layout: docs
 type: markdown
 ---
 
-The page explains Istio's overall service model.
-
-
-
-The Istio manager serves as an interface between the user and Istio,
-collecting configuration, validating it and propagating it to various
-components. It abstracts platform-specific implementation details from the
-mixer and proxies, providing them with an abstract representation of user's
-services that is independent of the underlying platform. The model
-described below assumes a microservice-based application owned by a single
-tenant.
-
-**Istio-managed service:** Modern cloud applications are created by
-composing together independent sets of services. Individual services are
-referenced by their fully-qualified domain name (FQDN) and one or more
-ports where the service is listening for connections.
-
-**Platform-agnostic internal representation:** The Istio model of a
-microservice is independent of how it is represented in the underlying
-platform (Kubernetes, Mesos, CloudFoundry, etc.). Platform-specific
-adapters are responsible for populating the internal model representation
-with various fields, from the metadata found in the platform.
+The Istio model of a microservice is independent of how it is represented
```

Comment:
I agree. Better to remain consistent and independent of marketing lingo. Will change the docs.